packages/web3.js-plugin-fee-estimation/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "@eth-optimism/web3.js-plugin-fee-estimation",
+  "version": "0.1.0",
+  "description": "A Web3.js plugin for doing OP-Chain gas estimation",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ethereum-optimism/optimism.git",
+    "directory": "packages/web3js-plugin-fee-estimation"
```

```suggestion
    "directory": "packages/web3.js-plugin-fee-estimation"
```
Missing a '.'

FWIW I would slightly prefer removing the '.' from the file path but that's just a nit up to you